README.md

```diff
@@ -1,10 +1,34 @@
-- 👋 Hi, I’m @loydpadgett
-- 👀 I have experience with PowerCLI, VEEAM Backup and Replication, Ansible Core, SNMP, GIT, firmware
-- 👁️ I also have experience with Nagios, Nagvis, Prometheus, LAMP stack, powershell, bash, (g)awk, authentication modules
-- 🌱 I’m currently using GIT, PowerCLI, nginx, vsCode, writing authentication modules for encryption
-- 💞️ I’m looking to collaborate on Powershell, PowerCLI, Ansible Core and Nginx
-- 💌 I also have experience developing microcontrollers and building analog circuitry. 
-- 📫 How to reach me loydpadgett@gmail.com
+### **👋 Hi, I’m @loydpadgett**
+#### **👀 I have experience with** 
+ ### System Administration
+   - **PowerCLI**
+   - **VEEAM Backup and Replication Powershell module**
+   - **Ansible Core** 
+   - **Printer Management**
+   - **bash** 
+   - **powershell**
+ ### Asset Monitoring   
```

Comment:
I like this format better, I not have it on vsCode so I can change quickly in vim